docs/architecture/modernize-desktop/deploy-modern-applications.md

```diff
@@ -0,0 +1,220 @@
+
+![MSIX Packaging Tool](media\deploying-modern-desktop-applications\msix-packaging-tool.png)
+
+#### Package Support Framework
+
+The Package Support Framework is an open-source kit that helps you apply fixes to your existing Win32 application when you don't have access to the source code, so that it can run in an MSIX container. The Package Support Framework helps your application follow the best practices of the modern runtime environment.
+
+#### App Installer
+
+App Installer allows Windows 10 apps to be installed by double-clicking the app package. This means that users don't need to use PowerShell or other developer tools to deploy Windows 10 apps. The App Installer can also install an app from the web, optional packages, and related sets.
+
+## How to create an MSIX package from an existing Win32 desktop application
+
+Let's go through the process to create an MSIX package from an existing Win32 application. In this example, we'll use a Windows Forms app.
+
+To start, add a new project to your solution, select the Windows Application Packaging Project and give it a name.
+
+![Adding new Windows Application Packaging Project](media\deploying-modern-desktop-applications\adding-packaging-project.png)
+
+You'll see the structure of the packaging project and note a special folder called Applications. Inside this folder you can specify which applications you want to include in the package, it can be more than one.
+
+![The structure of the packaging project](media\deploying-modern-desktop-applications\packaging-project.png)
+
+Right-click on the *Applications* folder and select the Windows Forms project you want to package from the Visual Studio solution.
+
+![Adding our Windows Forms project to Packaging Project](media\deploying-modern-desktop-applications\adding-our-project.png)
+
+At this point, you can compile and generate the package but let's take a look at a couple of things. To have a better user experience, Visual Studio can autogenerate all the visual assets a modern application needs to handle icons and tile assets for the tile bar and start menu. Opening the *Package.appxmanifest* file you can access the Manifest Designer and generate all the visual assets from a given image present on your project just by clicking *Create*.
+
+![Manifest Designer](media\deploying-modern-desktop-applications\manifest-designer.png)
+
+If you open the code for the *Package.appxmanifest* file, you can see a couple of interesting things.
+
+Right under `<Package>` there is an `<Identity>` node. This is where your packaged application is going to get its identity to let the OS manage it.
+
+![Identity node](media\deploying-modern-desktop-applications\identity-node.png)
+
+In the `<Capabilities>` node you can find all the requirements the application express to need, paying special attention to the `<rescap:Capability Name="runFullTrust" \>` that tells the OS to run the app in full trust mode as this is a Win32 application.
```

Comment:
```suggestion
In the `<Capabilities>` node, you can find all the requirements the application needs, paying special attention to the `<rescap:Capability Name="runFullTrust" \>`, which tells the OS to run the app in full trust mode since it's a Win32 application.
```